Update machete to Gradle 5.6.2, and flare multi-module library plug-in

romeara opened the pull request.

romeara committed:
Update machete to Gradle 5.6.2, and flare multi-module library plug-in

ghost commented:
## Black Duck Security Report

Merging #16 into master will increase security risk!

### Added Components

Clean:       5

### Removed Components

Clean:       6

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/StarChart-Labs/machete/pull-requests/16)

---

2 component(s) did not change but have updated security data. This may affect your calculated score.

romeara committed:
Make root project name static regardless of git

romeara committed:
Remove module name change, converting to a GitHub issue (GH-17)

There was an error in the automatic module name generation used for
machete jars due to a copy-paste issue. As this is not a
backwards-compatible change, file this as an issue so the fix timing may
be controlled and done only when a major version release is made.
